Mixpanel/Mixpanel.m

```diff
@@ -45,8 +45,12 @@ + (Mixpanel *)sharedInstanceWithToken:(NSString *)apiToken launchOptions:(NSDict
 #else
     const NSUInteger flushInterval = 60;
 #endif
+    
+    return [[self alloc] initWithToken:apiToken launchOptions:launchOptions flushInterval:flushInterval trackCrashes:trackCrashes automaticPushTracking:automaticPushTracking];
+}
 
-    return [[self alloc] initWithToken:apiToken launchOptions:launchOptions andFlushInterval:flushInterval];
++ (Mixpanel *)sharedInstanceWithToken:(NSString *)apiToken launchOptions:(NSDictionary *)launchOptions {
+    return [Mixpanel sharedInstanceWithToken:apiToken launchOptions:launchOptions trackCrashes:YES automaticPushTracking:NO];
```

Comment:
To keep behavior the same as before, let's make `automaticPushTracking:YES` default

Reply:
Yes, my bad